cla-backend-go/v2/dynamo_events/github_repository.go

```diff
@@ -13,76 +13,31 @@ import (
 	"github.com/aws/aws-lambda-go/events"
 	log "github.com/communitybridge/easycla/cla-backend-go/logging"
 	"github.com/communitybridge/easycla/cla-backend-go/utils"
-	v2ProjectService "github.com/communitybridge/easycla/cla-backend-go/v2/project-service"
 )
 
-// GithubRepoAddedEvent github repository added event
-func (s *service) GithubRepoAddedEvent(event events.DynamoDBEventRecord) error {
+// GithubRepoModifyEvent github repository modify event
+func (s *service) GithubRepoModifyAddEvent(event events.DynamoDBEventRecord) error {
+	ctx := utils.NewContext()
 	f := logrus.Fields{
-		"functionName": "GithubRepoAddedEvent",
+		"functionName":   "GithubRepoModifyEvent",
+		utils.XREQUESTID: ctx.Value(utils.XREQUESTID),
 	}
 
-	log.WithFields(f).Debug("GithubRepoAddedEvent called")
 	var newRepoModel repositories.RepositoryDBModel
 	err := unmarshalStreamImage(event.Change.NewImage, &newRepoModel)
```

Comment:
event.Change.NewImage will be empty for Remove events.  We need to check the even type, if New/Updated, then use the NewImage, if Removed, then use OldImage.

Reply:
1. eventName = "INSERT" or eventName = "MODIFY"  => unmarshall the newImage
1. eventName = "REMOVE"  => unmarshall the oldImage